dev/run-tests.py

```diff
@@ -567,7 +620,11 @@ def main():
 
     modules_with_python_tests = [m for m in test_modules if m.python_test_goals]
     if modules_with_python_tests:
-        run_python_tests(modules_with_python_tests, opts.parallelism)
+        # We only run PySpark tests with coverage report in one specific job with
+        # Spark master with SBT in Jenkins.
+        is_sbt_master_job = "SPARK_MATER_SBT_HADOOP_2_7" in os.environ
```

Comment:
typo: `MASTER` not `MATER`.  (I guess then this also requires @shaneknapp to make the corresponding change to keep things aligned?)

Reply:
Argh, thanks for finding this.. let me double check the env and fix it accordingly. Probably my mistake.

Reply:
Shane, I made a change in Jenkins configuration (`SPARK_MATER_SBT_HADOOP_2_7` -> `SPARK_MASTER_SBT_HADOOP_2_7`) since it;s just a typo fix.